src/app/plan/components/Wizard/HooksStep.module.scss

```diff
@@ -0,0 +1,8 @@
+@import "src/colors";
+
+.formContainerStyles {
```

Comment:
Do we really need these? the margin and padding can probably be done with the PF spacer utility classes, and I'm not sure we need the light gray border around the form. @vconzola how do you feel about that grey box? I'm assuming it was added because there was a box around the form in your mocks, but was that just a structural thing or did you intend for it to have a border? I don't think any of our other forms have this kind of border.

Reply:
I will remove the custom styles here.